src/sentry/tasks/code_owners.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from sentry import features
+from sentry.tasks.base import instrumented_task
+
+logger = logging.getLogger("sentry.tasks.code_owners")
+
+
+@instrumented_task(
+    name="sentry.tasks.update_code_owners_schema",
+    queue="code_owners",
+    default_retry_delay=5,
+    max_retries=5,
+)
+def update_code_owners_schema(organization=None, integration=None, projects=None, **kwargs):
```

Comment:
we can change the arguments to always expect an organization (and not have the default be `None`)